docs/asciidoc/http-client.adoc

```diff
@@ -16,22 +17,10 @@ To connect the `HTTP` client to a given `HTTP` endpoint, you must create and con
 The following example shows how to do so:
 
 ====
-[source,java]
+[source,java,indent=0]
+./../../reactor-netty-examples/src/main/java/reactor/netty/examples/documentation/http/client/connect/Application.java
```

Comment:
this seems redundant with what's below. can't it use the `{examplesdir}` as well ?

Reply:
I wasn't able to find a way to do that. Do you know how it can be done?

Reply:
scratch that, found it

Reply:
done